### PR TITLE
Added M&S case study

### PR DESCRIPTION
**Pull request by MiyamotoAkira**

### CODE REVIEW

File: src/clients/projects/software-craftsmanship-at-mands.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: case_study
+title: Software Craftsmanship at M&S
+identifier: MANDS
+company-name: ASOS
```
Comment (upsd) [inline src/clients/projects/software-craftsmanship-at-mands.html]: `ASOS`?

File: src/_i18n/en.yml [hunk 2]
```diff
@@ -629,3 +629,13 @@ projects:
                      <p>Part of the feature set included a reporting tool to provide the business with detailed reports on the event. We spent time to help train the AWS marketing team on how to use the system.</p>
     outcome: <p>Given the tight feedback loops and real-time interactions with the Product Owner, we were able to deliver both the MVPs for the project and several additional features ahead of the deadline. The event team were able to respond quickly to demand via the real-time data analysis (for instance session scheduling, staff rotas) and report session take-up and popularity to stakeholders.</p>
              <p>AWS invited Codurance to add additional features to the system so that it could be used at future events. Our team also gave a talk on Software Craftsmanship at the AWS Summit London 2016.</p>
+  MANDS:
+    link-text: Case study at M&S
+    short_description: We helped transition their processes to cater for new projects
```
Comment (alfredodev) [inline src/_i18n/en.yml:634]: I just realised on the company page that we have a mix of separators `-` vs `_` and is the same here

File: src/clients/projects/software-craftsmanship-at-mands.html (hunk 1 above)
Comment (alfredodev) [inline src/clients/projects/software-craftsmanship-at-mands.html]: wrong company name

File: src/_i18n/en.yml [hunk 3]
```diff
@@ -629,3 +629,13 @@ projects:
...
+    long_description: We provided mentoring and support at M&S, while there was a cultural transformation helping them prepare for their new projects and environment.
+    client_description: Marks and Spencer is a major British multinational retailer, specialising in the selling of clothing, home products and luxury food products.
+    the_challenge: <p>Marks and Spencers’ engineering department was entering a period of cultural change, with people being re-assigned to new teams ahead of several large digital projects.
```
Comment (alfredodev) [inline src/_i18n/en.yml:637]: Can we remove the < p > from the translations and put them on the page?

File: src/clients/projects/software-craftsmanship-at-mands.html (hunk 1 above)
Comment (MiyamotoAkira) [reply src/clients/projects/software-craftsmanship-at-mands.html]: Ooopsies

File: src/_i18n/en.yml [hunk 4]
```diff
@@ -629,3 +629,13 @@ projects:
...
              <p>AWS invited Codurance to add additional features to the system so that it could be used at future events. Our team also gave a talk on Software Craftsmanship at the AWS Summit London 2016.</p>
+  MANDS:
+    link-text: Case study at M&S
+    short_description: We helped transition their processes to cater for new projects
+    long_description: We provided mentoring and support at M&S, while there was a cultural transformation helping them prepare for their new projects and environment.
+    client_description: Marks and Spencer is a major British multinational retailer, specialising in the selling of clothing, home products and luxury food products.
+    the_challenge: <p>Marks and Spencers’ engineering department was entering a period of cultural change, with people being re-assigned to new teams ahead of several large digital projects. 
+    our_sc_approach: <p>With our connections to the wider community and passion for Software Craftsmanship, Codurance were seen as a partner could help to raise both code quality and to help establish a new team with this department through mentoring and guidance.
```
Comment (alfredodev) [inline src/_i18n/en.yml]: I think there is something missing in `Codurance were seen as a partner could help to raise both code quality `, that could help?

File: src/_i18n/en.yml (hunk 3 above)
Comment (alfredodev) [reply src/_i18n/en.yml:637]: maybe not...

File: src/clients/projects/software-craftsmanship-at-mands.html (hunk 1 above)
Comment (MiyamotoAkira) [reply src/clients/projects/software-craftsmanship-at-mands.html]: Late. Fixed already :-)

File: src/_i18n/en.yml (hunk 3 above)
Comment (MiyamotoAkira) [reply src/_i18n/en.yml:637]: There was a reason they are over there. But can't remember why. Something with the way it is displayed, or was displayed.

File: src/_i18n/en.yml (hunk 2 above)
Comment (MiyamotoAkira) [reply src/_i18n/en.yml:634]: Will take care of that as a separate branch/PR

**Review by upsd — APPROVED** [review]
LGTM

**Pull request merged**